src/Morpho.sol

```diff
@@ -1,7 +1,16 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.19;
 
-import {Id, IMorpho, MarketParams, Position, Market, Authorization, Signature} from "./interfaces/IMorpho.sol";
+import {
+    Id,
+    IMorphoStaticTyping,
+    IMorphoBase,
```

Comment:
why do we import IMorphoBase ? doesn't IMorphoStaticTyping inherit from IMorphoBase?

Reply:
yes it does but the compiler does not allow to use `IMorphoStaticTyping` only...